content/pages/conference/2024/03-workshop.rst

```diff
@@ -38,6 +38,13 @@ Agneda
 About the workshop
 ------------------
 
+We will invite professionals in numerical computing or software engineering relevant field to share their development
```

Comment:
It will be good if it adds only the topic, like:

> **Topic**: Advanced CFD software development

If you want to add the passage, it may be put along with the texts after the topic and speaker.